demo/js/application.js

```diff
@@ -4,52 +4,165 @@ let readline = require('readline-sync');
 
 class Application {
     start(){
-        let printer = new Printer();
+        this._printer = new Printer();
+        this._library = new Library();
 
-        printer.basicMessage('Welcome to this library demo!');
+        this._printer.basicMessage('Welcome to this library demo!');
         while(true){
             this.startingChoice();
         }
     }
 
     startingChoice(){
-        let printer = new Printer();
-        let library = new Library();
-
-        printer.paragrapheStartMessage('Please choose a number to pick an option : ');
-        printer.basicMessage('L - List all books in the library.');
-        printer.basicMessage('A - Add a book to the library.');
-        printer.basicMessage('U - Update a book of the library.');
-        printer.basicMessage('D - Delete a book of the library.');
-        printer.basicMessage('exit - Leave de application');
-
-
+        this._printer.paragrapheStartMessage('Please choose a letter to pick an option : ');
+        this._printer.basicMessage('L - List all books in the library.');
+        this._printer.basicMessage('A - Add a book to the library.');
+        this._printer.basicMessage('U - Update a book of the library.');
+        this._printer.basicMessage('D - Delete a book of the library.');
+        this._printer.basicMessage('exit - Leave de application');
 
         let response = '';
         response = readline.question('');
 
         switch(response){
         case 'L':
-            library.listBookOfLibrary();
+            this._library.listBookOfLibrary();
             break;
         case 'A':
-            printer.basicMessage('A');
+            this.addBookChoice();
             break;
         case 'U':
-            printer.basicMessage('U');
+            this.updateBookChoice();
             break;
         case 'D':
-            printer.basicMessage('D');
+            this.deleteBookChoice();
             break;
         case 'exit':
             process.exit(0);
             break;
         default:
-            printer.warningMessage('your choice is not an option');
+            this._printer.warningMessage('your choice is not an option');
             return false;
         }
         return true;
     }
```

Comment:
Could be a map instead of a switch-case as in
```
let optionsMap : {
  'A' : this.addBookChoice.bind(this),
  'U' : this.updateBookChoice.bind(this),
  'exit' : process.exit
}
if(!response in Object.keys(optionsMap)){
  return false;
}
optionsMap[response]();
```